cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-option-is-not-available.cy.js

```diff
@@ -0,0 +1,78 @@
+import TopMenu from '../../../support/fragments/topMenu';
+import TestTypes from '../../../support/dictionary/testTypes';
+import Permissions from '../../../support/dictionary/permissions';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import DevTeams from '../../../support/dictionary/devTeams';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import FileManager from '../../../support/utils/fileManager';
+import Users from '../../../support/fragments/users/users';
+
+let user;
+const item = {
+  instanceName: `testBulkEdit_${getRandomPostfix()}`,
+  itemBarcode: getRandomPostfix(),
+  instanceId: '',
+  holdingUUID: '',
+  holdingHRID: '',
```

Comment:
do you need these instantiated here? instanceId: '',
  holdingUUID: '',
  holdingHRID: '',

Reply:
instanceId and  holdingUUID I`ve used, only holdingHRID doesnt need!